src/main/java/igym/config/ValidationConfig.java

```diff
@@ -13,7 +13,7 @@ public class ValidationConfig {
     static public Validator validator() {
         ValidatorFactory factory = Validation.byDefaultProvider()
                 .configure()
-                .addProperty("hibernate.validator.fail_fast", "true")  
+                .addProperty("hibernate.validator.fail_fast", "false")  
                 .buildValidatorFactory();
         return factory.getValidator();
     }
```

Comment:
Is this file still necessary? Because we are not using this set up anymore. Could you please let me know what you think about removing this file?